bigtable/client/filters_test.cc

```diff
@@ -0,0 +1,76 @@
+// Copyright 2017 Google Inc.
```

Comment:
These tests are interesting in that the implementation and the test test code (the `EXPECT`s) are almost exactly the same, so seems kind of likely that a mistake in one would end up in the other. The Go client tests filter facade implementations against the emulator, basically just making sure behavior of the filter is correct vs checking the proto fields. Something to consider for later...

Reply:
Agree.